.hooks/pre-commit

```diff
@@ -7,7 +7,7 @@ if [ $fmtcount -gt 0 ]; then
 fi
 
 # Due to the way composites work, vet will fail for some of our tests so we ignore it
-vetcount=`go tool vet --composites=false ./ 2>&1  | wc -l`
+vetcount=`go tool vet --composites=false --methods=false ./ 2>&1  | wc -l`
```

Comment:
Let me know if you have a better method name than Seek().

It probably makes sense to move to iterators (with just a Next() method) instead of cursors but that's a refactor for another time.